sale_order_lot_generator/models/sale_order_line.py

```diff
@@ -21,21 +21,21 @@ def _prepare_vals_lot_number(self, index_lot):
     def create_prodlot(self, index_lot=1):
```

Comment:
To be consistent I supose create_prodlot should be rename without lot

Reply:
Done

Reply:
Done

Reply:
a detail, but if you changed create_prodlot to create_lot, I would change also the generate_prodlot method to generate_lot. If you do it I approve right now...

Reply:
Done